clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -1646,9 +1646,38 @@ private DescribeQuorumRequest createDescribeQuorumRequest(short version) {
         return new DescribeQuorumRequest.Builder(data).build(version);
     }
 
-    private DescribeQuorumResponse createDescribeQuorumResponse() {
+    private DescribeQuorumResponse createDescribeQuorumResponse(short version) {
+        DescribeQuorumResponseData.ReplicaState replicaState = new DescribeQuorumResponseData.ReplicaState()
+                .setReplicaId(1);
+        if (version >= 1) {
+            replicaState.setLastFetchTimestamp(123L)
+                .setLastCaughtUpTimestamp(123L);
+        }
+        if (version >= 2) {
+            replicaState.setReplicaDirectoryId(Uuid.randomUuid());
+        }
+        DescribeQuorumResponseData.PartitionData partitionData = new DescribeQuorumResponseData.PartitionData()
+                .setPartitionIndex(0)
+                .setCurrentVoters(singletonList(replicaState))
+                .setObservers(singletonList(replicaState));
+        DescribeQuorumResponseData.TopicData topicData = new DescribeQuorumResponseData.TopicData()
+                .setTopicName("topic1")
```

Comment:
nit: should we just make this `__cluster_metadata`

Reply:
Thanks, fixed